### PR TITLE
React Capstone Project: World Metric App

### PR DESCRIPTION
**Pull request by theodoku**
In this Pull Request:
**1. Select API for the Project.**
**2. Build files structure for project.**
**3. Prepare routes and navigation for app.**
**4. Created a function to sure that a user can display a list of items and filter them and get data from the API.**
**5. Style  components to match the design provided.**
**6. Create the tests for the application.**
**7. Record a video for  project**
**8. Create a  README file.**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,212 @@
...
+<!-- AUTHORS -->
+## 👥 Authors <a name="authors"></a>
+
+ The collaborators of this project.
+
+👤 **theodoku**
+
+- GitHub: [@githubhandle](https://github.com/theodoku)
+- Twitter: [@twitterhandle](https://twitter.com/dok_theo)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/Theophilusdoku)
```
Comment (abdulhamiid) [inline README.md]: Currently, the link to your `LinkedIn` profile is broken. Please kindly fix this, 

Let's avoid a broken link in our readme file for a professional repository. 🛠👍🤝

image

**Review by abdulhamiid — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:

Hi @theodoku 👋, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>



## Suggested changes ♻️
_Please kindly check the comments under the review._

Your commit messages are meaningful 👍 💯 However, According to [Professional repo - basic rules](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) The commit message should be written in the imperative tense and capitalized. 

This means we need to eliminate the temptation to use gerunds or past tense in our subject lines. 🛠

Your commits 👀

image

Good commits 🛠
```
- Add Dashboard.js, Header.js and installed react-icons
- Modify Dashboard.js and Header.js and Add wrapper.js, Header.css
- Modify Header.js, wrapper.js, Dashboard.js, add pages folder, sto…
```

- [ ] **IMPORTANT:** As it is not easy to change commit history, kindly correct this behavior in the future. 🤗💪

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: README.md (hunk 1 above)
Comment (theodoku) [reply README.md]: Broken Link fixed and updated🙌

**Review by jerryowusu — APPROVED** [review]
Hi @theodoku  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

CookedDelectableDolphin-size_restricted

## Highlights
:heavy_check_mark: LinkedIn link updated 
:heavy_check_mark: Great effort so far 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**